src/scripts/main.js

```diff
@@ -1,3 +1,578 @@
 'use strict';
 
 // write your code here
+
+const start = document.querySelector('.start');
+const gameBoard = document;
+const cells = document.querySelectorAll('.field_cell');
+const messageStart = document.querySelector('.message_start');
+const messageWin = document.querySelector('.message_win');
+const scoreBoard = document.querySelector('.game-score');
+let score = 0;
+
+let arrayCells = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+];
+
+const cellToRemove = [
+  'field_cell--2',
+  'field_cell--4',
+  'field_cell--8',
+  'field_cell--16',
+  'field_cell--32',
+  'field_cell--64',
+  'field_cell--128',
+  'field_cell--256',
+  'field_cell--512',
+  'field_cell--1024',
+  'field_cell--2048',
+];
+
+start.addEventListener('click', function() {
+  if (this.classList.contains('start')) {
+    this.textContent = 'Restart';
+
+    this.classList.remove('start');
+    this.classList.add('restart');
+    messageStart.classList.add('hidden');
+
+    addRandomValues();
+    addRandomValues();
+  } else {
+    this.textContent = 'Start';
+
+    this.classList.remove('restart');
+    this.classList.add('start');
+    messageWin.classList.add('hidden');
+    messageStart.classList.remove('hidden');
+
+    cleaner();
+  }
+});
+
+function randomizer(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function addRandomValues() {
+  const value = randomizer(0, cells.length - 1);
+  let abc;
+
+  const val = value % 4;
+
+  if (value <= 3) {
```

Comment:
The conditional chain from line 66 to 73 is too complex. It's advisable to use a more readable and maintainable approach. Consider creating a function for this operation.